libr/debug/p/debug_native.c

```diff
@@ -338,75 +337,14 @@ static RList *r_debug_native_pids (int pid) {
 			if (p) r_list_append (list, p);
 		}
 	}
+#elif __linux__
+	return linux_pid_list (pid, list);
 #else
-	int i, fd;
```

Comment:
this is suposed to be the implementation for BSD systems. and it was working iirc, dont remove it, just move it into a separate function, which will fail in case there's no /proc mounted